content/en/admin/scaling.md

```diff
@@ -86,48 +86,48 @@ Many tasks in Mastodon are delegated to background processing to ensure the HTTP
 
 #### Number of threads {#sidekiq-threads}
 
-While the amount of threads in the web process affects the responsiveness of the Mastodon instance to the end-user, the amount of threads allocated to background processing affects how quickly posts can be delivered from the author to anyone else, how soon e-mails are sent out, etc.
+While the number of threads in the web process affects the responsiveness of the Mastodon instance to the end-user, the number of threads allocated to background processing affects how quickly posts can be delivered from the author to anyone else, how soon e-mails are sent out, etc.
 
 The number of threads is not regulated by an environment variable, but rather through a command line argument when invoking Sidekiq, as shown in the following example:
 
 ```bash
 bundle exec sidekiq -c 15
 ```
 
-This would initiate the Sidekiq process with 15 threads. It's important to note that each thread requires a database connection, necessitating a sufficiently large database pool. The size of this pool is managed by the DB_POOL environment variable, which should be set to a value at least equal to the number of threads.
+This would initiate the Sidekiq process with 15 threads. It is important to note that each thread requires a database connection, so this requires a large database pool. The size of this pool is managed by the `DB_POOL` environment variable, which should be set to a value at least equal to the number of threads.
 
 #### Queues {#sidekiq-queues}
 
 Sidekiq uses different queues for tasks of varying importance, where importance is defined by how much it would impact the user experience of your server’s local users if the queue wasn’t working, in order of descending importance:
 
 | Queue       | Significance                                                                                                                         |
```

Comment:
this would probably be better as either h5 (#####) or a definition list (below)

```
`queue_name`
: definition of the queue
```

Reply:
I'm not a huge fan of tables in Markdown, and this looks cleaner, so happy to accept this suggestion!